docs/network/index.html

```diff
@@ -1323,6 +1323,24 @@ <h2 id="Events">Events</h2>
   }
 }
 </pre>
+
+This is the structure common to all events. Specifically for the click event, the following property is added:
+
+<pre class="prettyprint lang-js">{
+...
+  items: [Array of click items],
+}</pre>
+
+Where the click items can be:
+<pre class="prettyprint lang-js">
+  {nodeId:number}            // node with given id clicked on
+  {nodeId:number labelId:0}  // label of node with given id clicked on
+  {edgeId:number}            // edge with given id clicked on
+  {edge:number, labelId:0}   // label of edge with given id clicked on
```

Comment:
Can't edge and node IDs be strings as well?

Reply:
Indeed. Would a `@typedef` be acceptable here? e.g.

```
@typedef {number|string} Id
```
or

```
@typedef {number|string} NodeId
@typedef {number|string} NdgeId
@typedef {number|string} LabelId
```

Reply:
Imo, a top level typedef `@typedef {number|string} Id` and then
```
@typedef {Id} NodeId
@typedef {Id} EdgeId
@typedef {Id} LabelId
```
But maybe that's needless indirection.

Personally I like it, because it's descriptive and flexible for the future.

Reply:
I like it too; the only qualm that I have about it, is that it will duplicate some text within the definitions, e.g.

```
@typedef {{nodeId:NodeId}} param
```
Actually, that isn't so bad. I'm actually inclined to go with this. Will let it simmer a bit to see if I'm going with this.

Reply:
OK, I'm won over. It's the most complete definition for the id's.